brave/browser-laptop
# Add clearAutocompleteData & clearAutofillData to clearBrowsingDataPanel and clear on shutdown

fix #3458
required https://github.com/brave/electron/pull/53

Auditors: @bridiver, @bbondy

Test Plan:
For autocomplete data in clearBrowsingDataPanel:
1. Go to site contains form and submit form after input some value
2. Double click the input field to make sure it popup the value you previous input
3. Go to "about:preferences#security" and click "Clear Browsing Data Now..."
4. Toggle on "Autocomplete data" and click "Clear"
5. Go previous site you submitted form
6. Double click the input field should popup nothing

For autocomplete data clear on shutdown:
1. Go to site contains form and submit form after input some value
2. Double click the input field to make sure it popup the value you previous input
3. Go to "about:preferences#security" and toggle on "Autocomplete data"
4. Quit brave and relaunch it
5. Go previous site you submitted form
6. Double click the input field should popup nothing

For autofill data in clearBrowsingDataPanel:
1. Add address and credit card in "about:autofill"
2. Go to the site you usually verify autofill and verify it
3. Go to "about:preferences#security" and click "Clear Browsing Data Now..."
4. Toggle on "Autofill data" and click "Clear"
5. Go previous site you verify autofill
6. Double click the input field should popup nothing

For autofill data clear on shutdown:
1. Add address and credit card in "about:autofill"
2. Go to the site you usually verify autofill and verify it
3. Go to "about:preferences#security" and toggle on "Autofill data"
4. Quit brave and relaunch it
5. Go previous site you verify autofill
6. Double click the input field should popup nothing
